lib/Listener/RegisterOperationsListener.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2024 Arthur Schiwon <blizzz@arthur-schiwon.de>
```

Comment:
nit: spdx to make Andy happy

Reply:
thought about that, but the remainder of the app is not. consistency? :grin:

Reply:
Fair point. Then it might be better to leave the old style and convert later :+1: